colors/dracula.vim

```diff
@@ -237,10 +241,15 @@ call s:h('Normal', s:fg, s:Background())
 call s:h('StatusLine', s:none, s:bglighter, [s:attrs.bold])
 call s:h('StatusLineNC', s:none, s:bglight)
 call s:h('WildMenu', s:bg, s:purple, [s:attrs.bold])
-call s:h('CursorLine', s:none, s:subtle)
 
-hi! link ColorColumn  DraculaSelection
-hi! link CursorColumn DraculaSelection
+if g:dracula_cursorline
```

Comment:
I'm not sure I follow why we conditionally define `CursorLine`; I thought the idea was to define it to Dracula's colors and let user's decide whether or not they turn it on. I've recently started working without it entirely, but this change doesn't make any sense to me.

Effectively, this feels weird and non-intuitive:

1. `let g:dracula_cursorline = 0` in vimrc
2. Start vim
3. Turn on cursorline, expecting to get a (probably non-Dracula) cursorline... and there isn't one? It's just... gone?

Without highlighting, it almost doesn't matter whether I have it on or off (and it seems like we're implicitly making that decision for the user).

I'd rather just leave the highlighting the way it is, and let the user clear it if they want, turn off/on if they want, etc. 

Let me know if my thoughts here are confusing; I'm finding it hard to express quite why I don't like this. It just seems like setting that variable disables cursorline entirely.

Reply:
Nope, your thoughts are valid.

The reason for the conditional definition here is **exclusively** due to the fact that some people (me) want a situation where the cursor line is not highlighted, **BUT** the line number of the cursor line is highlighted in all modes. That only works by doing it this way. (see `:h hl-CursorLineNr` for more info)

Reply:
Ahh. I see... hm. Can that be written somewhere besides this issue to tie it in better as justification? Have we thought about different solutions? Could we perhaps add a note to the documentation for users that might want e.g.:

```vim
autocmd ColorScheme dracula hi clear CursorLine
```

to solve the problem? This seems like a non-dracula problem to me regardless.

I use relative number so I wasn't aware of that problem (the highlighting is always active with relativenumber on).

Reply:
^^^I like that idea muuuuuuuch better. I wasn't aware `ColorScheme` was an event. Good deal. I'll remove this stuff